### PR TITLE
add roadhog "hash" charater

### PR DESCRIPTION
**Pull request by happier2**
Add roadhog "hash" charater, by adding 'src/index.ejs' and '"hash": true' in './.roadhogrc'. In this way, roadhog emitting files in the './dist' directory would look like '[name]_[chunkhash].xxx'

### CODE REVIEW

File: src/index.ejs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Ant Design Pro</title>
+    <link rel="icon" href="https://gw.alipayobjects.com/zos/rmsportal/IOtlElCiWVIOZqgDslYd.png" type="image/x-icon">
+</head>
+
+<body>
+    <div id="root"></div>
+    <script src="https://gw.alipayobjects.com/as/g/??datavis/g2/2.3.12/index.js,datavis/g-cloud/1.0.2/index.js,datavis/g2-plugin-slider/1.2.1/slider.js"></script>
```
Comment (afc163) [inline src/index.ejs]: Use 2 space as indent.

File: src/index.ejs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+</body>
+
```
Comment (afc163) [inline src/index.ejs]: Remove these blanks

File: src/index.ejs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+</html>
```
Comment (afc163) [inline src/index.ejs]: Could we remove `index.html` now?

Comment (happier2) [reply src/index.ejs]: Yes, of course😄

**Comment by afc163** [comment]
will close #200

**Pull request merged**